app/src/main/java/fr/free/nrw/commons/modifications/ModificationsSyncAdapter.java

```diff
@@ -95,19 +95,24 @@ public void onPerformSync(Account account, Bundle bundle, String s, ContentProvi
 
                 Cursor contributionCursor;
                 try {
+                    //TODO: How do we prevent this NPE?
```

Comment:
return early if `contributionsClient` is null.

- also add a null check for getContext()

Reply:
Thanks, I have added the check for contributionsClient. :) However getContext() does not raise any possible NPE flags in Android Studio, is a null check really necessary whenever we call that?